tests/test_questions.py

```diff
@@ -472,6 +474,27 @@ def test_nested_checkboxes_question_followup_get_data(self):
             ('follow', 'a')
         ])) == {'lead': ['no', 'maybe not'], 'follow': None}
 
+    def test_get_error_messages_orders_by_question_order(self):
+        question = self.question().filter(self.context())
+
+        multi_question_form_errors = OrderedDict([
```

Comment:
Is the order of the form errors supposed to be important for this test? If so, could you explain why please (sorry, it's been a while since I've looked at the content loader).

Reply:
Yeah, I was trying to show that even if the errors came through to the method in a different order, the new method would sort by question order. However as you point out, the question order in the setup is alphabetical anyway :woman_facepalming: I'll fix that up for this test now.